pyam/core.py

```diff
@@ -1079,7 +1068,7 @@ def line_plot(self, x='year', y='value', **kwargs):
 
         see pyam.plotting.line_plot() for all available options
         """
-        df = self.join_meta(x=x, y=y, **kwargs)
+        df = self.as_pandas(with_metadata=kwargs)
```

Comment:
do we need to include `x` and `y` here? or no?

Reply:
I don't think we do - if `x` or `y` are variables, they will be included in the returned dataframe anyway. And I don't think we can do `x` or `y` as metadata columns currently?